owncloudApp/src/main/res/drawable/ic_baseline_lock_24.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I would rename the icon to do it more descriptive. For example `ic_lock`. And curiously, if you check it, we already have a lock icon 👍  Same for other icons, the more descriptive, the better, we will find them easily.